apps/backend/src/flat-files/tdi17/TDI17Details.ts

```diff
@@ -13,7 +13,7 @@ export interface ITDI17Details extends IFixedWidthRecord<ITDI17Details> {
   program_cd: string;
   deposit_date: string;
   transaction_type: number;
-  location: string;
+  location_id: number;
```

Comment:
Any reason why we switched to number, I think string seems more appropriate here?